docs/operating-scylla/nodetool-commands/scrub.rst

```diff
@@ -87,4 +109,64 @@ Scrub **a** specific table (mytable) in a keyspace (mykeyspace) in VALIDATE mode
 
    > nodetool scrub -m VALIDATE --no-snapshot mykeyspace mytable
 
+Scrub **a** specific table (mytable) in a keyspace (mykeyspace) in SEGREGATE mode dropping unfixable SSTables
+
+.. code-block:: shell
+
+   > nodetool scrub -m SEGREGATE --drop-unfixable-sstables mykeyspace mytable
+
+Procedures for Removing Bad SSTables
+.....................................
+
+Method 1: Quarantine and Drop
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Step 1**: Run scrub in VALIDATE mode to identify and quarantine corrupted SSTables:
+
+.. code-block:: shell
+
+   > nodetool scrub -m VALIDATE keyspace_name table_name
+
+This will move corrupted SSTables to a ``quarantine`` directory.
+The ``quarantine`` directory is a sub-directory of the table's respective data directory.
+
+**Step 2** (Optional): Preserve quarantined SSTables for analysis:
+
+Before permanently dropping the corrupted SSTables, consider copying all or a sample of them.
```

Comment:
... consider copying some or all of them aside, somewhere outside of the ScyllaDB data directory, so they are preserved for later investigation by the ScyllaDB R&D team, to determine the root cause of the corruption.